application/contract/member/member.go

```diff
@@ -17,16 +17,27 @@
 package member
 
 import (
-	"errors"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
 	"fmt"
-	"math"
+	"math/big"
+	"strconv"
+	"time"
 
+	"github.com/insolar/insolar/application/contract/member/helper"
 	"github.com/insolar/insolar/application/contract/member/signer"
+	"github.com/insolar/insolar/application/proxy/deposit"
+	"github.com/insolar/insolar/application/proxy/member"
 	"github.com/insolar/insolar/application/proxy/nodedomain"
 	"github.com/insolar/insolar/application/proxy/rootdomain"
 	"github.com/insolar/insolar/application/proxy/wallet"
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/logicrunner/goplugin/foundation"
+	"github.com/insolar/x-crypto/ecdsa"
+	"github.com/insolar/x-crypto/sha256"
+	"github.com/insolar/x-crypto/x509"
```

Comment:
No external imports (except for the platform itself) are allowed. If you need x-crypto, add it to the foundation lib.